tests/test_diffeq/test_odefiltsmooth/test_odefiltsmooth.py

```diff
@@ -167,13 +167,17 @@ def setUp(self):
 
     def test_kf_ibm1_stdev(self):
         """Standard deviation at end point roughly equal to tolerance."""
-        sol = probsolve_ivp(self.ivp, tol=self.tol, which_prior="ibm1", method="ekf0")
+        sol = probsolve_ivp(
+            self.ivp, atol=self.tol, rtol=self.tol, which_prior="ibm1", method="ekf0"
```

Comment:
I kept the delta small (out of laziness, honestly) by setting both atol and rtol to self.tol